aws-sagemaker-python/README.md

```diff
@@ -0,0 +1,73 @@
+# AWS SageMaker + Hugging Face LLM Deployment
+
+## Overview
+
+A Pulumi IaC program written in Python to deploy a Hugging Face Language Model (LLM) on AWS SageMaker.
+
+## Included:
+
+- IAM roles
+- SageMaker model endpoint
+- CloudWatch alarms
+
+## Prerequisites
+
+* Python 3.8+
```

Comment:
I would change this to "a supported version of Python" and link to the Python docs _unless_ Python 3.8 is needed (and based on the "Python 3.8+" I'd guess it isn't). 3.8 is nearing EOL.

Reply:
changed to 3.9 in https://github.com/pulumi/templates/pull/643/commits/974781ff9aa0bd4fa4983b5603a6410e31119843